test/integration_test.sh

```diff
@@ -80,80 +87,135 @@ ss2_version=${10}
 work_dir=$(pwd)
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# 1. Clone Lira if needed 
-# TODO: Create a repo for integration_test.sh to go in, so that we
-# can run in github mode. Currently we have to run in lira local mode.
-lira_dir=$script_dir/../
+# 1. Clone Lira if needed
+if [ $lira_mode == "github" ]; then
+  git clone git@github.com:HumanCellAtlas/lira.git
+  lira_dir=lira
+  cd $lira_dir
+  if [ $lira_version == "latest_released" ]; then
+    lira_version=$(python $script_dir/get_latest_release.py HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    lira_version=$(python $script_dir/current_deployed_version.py lira)
+  fi
+  git checkout $lira_version
+  cd -
+elif [ $lira_mode == "local" ]; then
+  lira_dir=$lira_version
```

Comment:
From the above comments, I think lira_version here is ignored?

Reply:
no longer being ignored, just pushed a new commit